src/Form/RulesSettingsForm.php

```diff
@@ -0,0 +1,76 @@
+<?php
+
+/**
+ * @file
+ * Contains \Drupal\rules\Form\RulesSettingsForm.
+ */
+
+namespace Drupal\rules\Form;
+
+use Drupal\Core\Form\FormStateInterface;
+use Drupal\Core\Form\ConfigFormBase;
+use Psr\Log\LogLevel;
+
+/**
+ * Provides rules settings form.
+ */
+class RulesSettingsForm extends ConfigFormBase {
+
+  /**
+   * {@inheritdoc}
+   */
+  public function getFormId() {
+    return 'rules_settings_form';
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  protected function getEditableConfigNames() {
+    return ['rules.settings'];
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function buildForm(array $form, FormStateInterface $form_state) {
+    $config = $this->config('rules.settings');
+
+    $form['log'] = [
+      '#type' => 'checkbox',
+      '#title' => $this->t('Log debug information to the system log'),
+      '#default_value' => $config->get('log'),
+    ];
+    $form['log_level'] = [
+      '#type' => 'radios',
+      '#title' => $this->t('Log level'),
+      '#options' => [
+        LogLevel::WARNING => $this->t('Log all warnings and errors'),
+        LogLevel::ERROR => $this->t('Log errors only'),
+      ],
+      '#default_value' => $config->get('log_level') ? $config->get('log_level') : LogLevel::WARNING,
+      '#description' => $this->t('Evaluations errors are logged to available loggers.'),
+      '#states' => [
+        // Hide the log_level radios when the debug log is disabled.
+        'invisible' => [
+          'input[name="log"]' => ['checked' => FALSE],
+        ],
+      ],
+    ];
```

Comment:
Do you want a checkbox there that act similiar as the log checkobox?

if ($this->config->get('log')) {
      if ($this->levelTranslation[$this->config->get('log_level')] >= $this->levelTranslation[$level]) {
        parent::log($level, $message, $context);
      }
    }